src/cargo/core/profiles.rs

```diff
@@ -552,9 +552,7 @@ fn merge_profile(profile: &mut Profile, toml: &TomlProfile) {
     }
     match toml.lto {
         Some(StringOrBool::Bool(b)) => profile.lto = Lto::Bool(b),
-        Some(StringOrBool::String(ref n)) if matches!(n.as_str(), "off" | "n" | "no") => {
-            profile.lto = Lto::Off
-        }
+        Some(StringOrBool::String(ref n)) if is_off(n.as_str()) => profile.lto = Lto::Off,
```

Comment:
I'm a little uncomfortable changing this to accept "none" as off, but in the grand scheme of things probably doesn't matter.

Reply:
to be honest, it made me uncomfortable too, but then I remembered that I copied/pasted this code in different places in the first iteration, and kinda decided to avoid that for future contributors.